.travis.yml

```diff
@@ -14,6 +14,11 @@ install: travis_retry composer install --prefer-source
 
 script: composer ci
 
+after_success:
+  - vendor/bin/phpunit --coverage-clover coverage.clover
```

Comment:
Doesn't this mean that tests are being run twice? Also, why not restrict this to one of the PHP versions (I suggest 7 as that's the fastest).